# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#goal definition
float32[] position
float32[] rate
float32 time
uint32 eeindex
uint32 home
float32 gripper_value
float32 gripper_rate
